langchain/base_language_model.py

```diff
@@ -0,0 +1,124 @@
+"""Base class for language models."""
```

Comment:
I had to move this here to avoid circular deps

Reply:
if we split up the schema file does that help at all?